### PR TITLE
Jingle/mui navigation

### PR DESCRIPTION
**Pull request by JingleH**

### CODE REVIEW

File: test/features/counter/Counter.spec.tsx [hunk 1]
```diff
@@ -6,134 +6,135 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
+//   it('should call incrementAsync after timer', () => {
+//     const fn = counterSlice.incrementAsync(1000);
+//     expect(fn).toBeInstanceOf(Function);
+//     const dispatch = jest.fn();
+//     // @ts-ignore
+//     fn(dispatch);
+//     jest.advanceTimersByTime(1001);
+//     expect(dispatch).toBeCalled();
+//   });
+// });
```
Comment (GalenWong) [inline test/features/counter/Counter.spec.tsx:140]: Delete all the commented out code. Just delete if we don't need them. The history are kept in git.

**Review by GalenWong — CHANGES_REQUESTED** [review]

File: test/features/counter/Counter.spec.tsx (hunk 1 above)
Comment (JingleH) [reply test/features/counter/Counter.spec.tsx:140]: Sure!

Comment (GalenWong) [reply test/features/counter/Counter.spec.tsx:140]: Don't forget the other files as well!

**Comment by GalenWong** [comment]
super fun home page haha. nice animation

**Pull request merged**